dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -299,6 +299,16 @@ META_DATA_EXCEPTION.useraction=Verify that the metadata is enabled on the backen
 # 7019 deleted
 # 7020 reserved for other NLS file
 # 7021-7042 deleted
+
+# Do not translate: java.util.Logging
+ORACLE_TRACE_PARSE_WARNING=DSRA7043W: A default value was used for the system property {0} because of parsing error: {1}
+ORACLE_TRACE_PARSE_WARNING.explanation=The system property strings for Oracle are parsed and passed to the java.util.Logging library.  If a property cannot be parsed a default value is used.
+ORACLE_TRACE_PARSE_WARNING.useraction=Examine the exception, and fix the DataSource configuration.
+
+ORACLE_TRACE_ENABLE_INFO=DSRA7044I: Some, but not all, of the required system properties [{0}] were configured to create custom Oracle trace file(s).
+ORACLE_TRACE_ENABLE_INFO.explanation=To configure custom Oracle trace file(s) all of the properties in this message need to be set as system properties.
+ORACLE_TRACE_ENABLE_INFO.useraction=If you do not want a custom Oracle trace file then this message can be ignored, otherwise, check your system properties and add the remaining properties.
```

Comment:
```suggestion
ORACLE_TRACE_ENABLE_INFO.useraction=If you want to create custom Oracle traces files, check your system properties and add the missing properties. Otherwise, you can ignore this message.
```